packages/connect-popup/src/index.tsx

```diff
@@ -13,10 +13,15 @@ import {
     PopupInit,
     PopupHandshake,
     MethodResponseMessage,
+    CORE_EVENT,
+    IFrameCallMessage,
     IFrameLogRequest,
     CoreMessage,
 } from '@trezor/connect/lib/exports';
+import { initLog } from '@trezor/connect/lib/utils/debug';
+import type { Core } from '@trezor/connect/lib/core';
 import { config } from '@trezor/connect/lib/data/config';
+import { parseConnectSettings } from '@trezor/connect-iframe/src/connectSettings';
```

Comment:
Just a note or question, probably not related to this PR, but shouldn't this `parseConnectSettings` be in connect-common?

Reply:
imho no. there are about 3 modules `parseConnectSettings`. each one is located to a place which suggests level of how strict this validation should be. in this case, it should be the same strength like when using iframe. so I decided to keep it there for now, but who knows